.circleci/config.yml

```diff
@@ -757,6 +757,10 @@ jobs:
           name: Run E2E tests
           command: yarn playwright-e2e
           working_directory: test-storybooks/portable-stories-kitchen-sink/react
+      - run:
+          name: Run Vitest
```

Comment:
why do we need this?

Reply:
ah yeah that's much better, it's to test these scenarios: https://github.com/storybookjs/storybook/pull/29826/files#diff-0550ef2a360c5242f25677cd964d3afce1c0412497a76eded7d03c0838941611R68-R92

Another alternative would be to move those stories into the UI Storybook instead, no strong opinion.